pkg/scheduler/objects/queue.go

```diff
@@ -1482,16 +1482,14 @@ func (sq *Queue) updateMaxResourceMetrics() {
 
 // updateAllocatedAndPendingResourceMetrics updates allocated and pending resource metrics if this is a leaf queue.
 func (sq *Queue) updateAllocatedAndPendingResourceMetrics() {
```

Comment:
please update comment also

Reply:
Sure @wilfred-s , sorry for missing this comment change.